### ISSUE TITLE
rtmp progress bar does not show

### ISSUE THREAD

**Issue by chad3814** [flowplayer/flash#208]
You can see an example of it in the demo: http://flash.flowplayer.org/plugins/streaming/rtmp.html

**Comment by phloxic** [comment]
Addendum: The reason is probably that one can see the full buffer (although for long movies it is so small, it's barely visible.
Flowplayer HTML5 on the contrary already has the probably more expected behaviour: the progress bar goes from 0 to the current playhead position.

**Comment by danrossi** [comment]
It's coded to remove the progress bar in the controls for rtmp. other providers set the start to begin at the seek location rather than stay at zero. All that would have to change I suppose. I guess this was to be able to display the back buffer bar correctly.

**Comment by phloxic** [comment]
Yes. The question is whether showing the rtmp buffer is useful/intuitive for the public or rather a technical thing (and strictly speaking correct for the name `bufferColor`, it does indeed show the current buffer), whereas the public just wants to see scrubber filled up to the current playhead position to get a more emphasized feedback of the current position relative to duration, the buffer is not of interest as they can seek to unbuffered positions (which is why showing the buffer for progressive download _is_ of interest for the public).
Personally I "work around" this by setting `bufferColor: "transparent"` in my setups.
One could argue that showing the actual buffer is more of interest for debugging, not for public consumption.

**Comment by chad3814** [comment]
I think the public is used to seeing both the buffer and the progress from the beginning to the playhead. Certainly you could show both, player.getStatus() has some buffer information even when using RTMP.

**Comment by phloxic** [comment]
@chad3814 - the buffer is minimal, and is already shown - you only see it if the movie is short enough, it would be even shorter if the back buffer was overlayed.
And displaying the buffer somehow suggests that it influences the user interaction but it does so only for progressive download.
But yeah, if both is possible.

**Comment by phloxic** [comment]
See also https://flowplayer.org/forum/#!/setup_configuration:progressbar-issue-in-flowpl

**Comment by danrossi** [comment]
there is two small changes required. one to check for rtmp in the controlbar and then changing some start flag back to zero and it will stay at the start. this gets affected after seeking or if you want the progressbar to move after seeking this can stay as is.

**Comment by phloxic** [comment]
See also: https://flowplayer.org/forum/#!/setup_configuration:video-doesnt-play-when-i-i
